lib/simple_kafka_consumer/consumer.rb

```diff
@@ -45,5 +45,14 @@ def debug(message)
     def consume(message)
       puts "doing nothing"
     end
+
+    def process(message)
+      meter(message)
```

Comment:
have you considered making this handle a block? so you could do something before or after the actual consuming? (or event get timing stats)